scene/gui/label.cpp

```diff
@@ -35,9 +35,15 @@
 
 void Label::set_autowrap(bool p_autowrap) {
 
+	const bool changed = (autowrap != p_autowrap);
+
 	autowrap = p_autowrap;
 	word_cache_dirty = true;
 	update();
+
+	if (changed && clip) {
```

Comment:
Shouldn't the `update()` call made at the end ?

Reply:
I'm not sure, but `set_clip_text` uses this order:

https://github.com/godotengine/godot/blob/0d2993659b1a6b4eaafcae55dde116e297170f3a/scene/gui/label.cpp#L556-L561

Reply:
hmm, maybe it doesn't matter. `update()` is supposed to trigger a redraw of the node, so I expected it to be at the end. But maybe minimum_size_changed() triggers an update too.

Reply:
Just checked the code. `minimum_size_changed` will result in a call to `update` if the size actually changed. (`_update_minimum_size` -> `_size_changed ` -> `item_rect_changed`)

Reply:
ok, sounds good then.